### ISSUE TITLE
Calcium oxalate casts

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5042]
Definition:The presence of birefringent calcium- and oxalate-containing casts
Modified definition: The presence of calcium- and oxalate-containing calculi (stones) in the kidneys.

**Comment by pnrobinson** [comment]
Is this different from 
https://hpo.jax.org/app/browse/term/HP:0020145 (Calcium oxalate crystalluria)?

I realize casts and crystals can be different in size but is this an important clinical distinction?

@avizrosenberg
@beckysteck
@yongqunh

**Comment by yongqunh** [comment]
This website says:
"Urine casts are cylindrical, cigar-shaped structures produced by the kidney and present in the urine in certain disease conditions." 
https://laboratoryinfo.com/types-of-casts-in-urine-and-their-clinical-significance/

This site says:
"Urine contains a large number of different chemicals. Under some circumstances, these chemicals may solidify into salt crystals. This is called crystalluria."
https://www.healthline.com/health/urine-crystals 
good ref: https://www.ncbi.nlm.nih.gov/pubmed/26509782 

more info in Wiki: 
https://en.wikipedia.org/wiki/Crystalluria 
https://en.wikipedia.org/wiki/Urinary_cast  

It appears that they are different more than size. Cellular casts appear not to be present in crystals. Crystal casts appear to be one that links both together. 
So uninary cast is a higher level term than urinary cast?  

@avizrosenberg
@beckysteck
@pnrobinson

**Comment by avizrosenberg** [comment]
I would interpret the Crystals in the HPO treatment as calculi which have a macroscopic definition. The crystals in the KPMP treatment are intended to reflect microscopic findings and can be either in a cast format (intratubular) or interstitial. While they can be found in patients with calculi, much of the time calculi ("kidney stones") are absent. Furthermore, they can reflect primary or secondary metabolic aberrations (i.e., oxaluria) which will be reflected existing HPO entries but are oft times more specific compared to the way we as histopathologists treat these findings and the way they will be collected in KPMP.

**Comment by pnrobinson** [comment]
@avizrosenberg
@beckysteck

From what I am reading, there are calcium oxalate "CRYSTALS" (not "casts"). Would this be the correct term label? If not, can you please suggest a PubMed reference for casts? Can we specify the difference between intratubular casts and crystals?

Calcium oxalate crystals withintubules
